.NET/Microsoft.Recognizers.Text.DateTime/Extractors/BaseTimePeriodExtractor.cs

```diff
@@ -291,5 +297,26 @@ private List<Token> MatchTimeOfDay(string text)
 
             return ret;
         }
+
+        // Support cases like "from 6 to 7" which are pure number ranges
+        // Only when the number range is at the end of a sentence, it will be considered as a time range
+        private List<Token> MatchPureNumberCases(string text)
+        {
+            var ret = new List<Token>();
+            foreach (var regex in this.config.SimpleCasesRegex)

```

Comment:
Why we use SimpleCasesRegex in this case? Should we have one specific Regex for the pure number case?